### ISSUE TITLE
Add Docker Environmental sections

### ISSUE THREAD

**Issue by johanels** [terraform-google-modules/terraform-google-container-vm#18]
Hi,

Could you maybe add a sample use of Docker env variables?

Thanks,

JE

**Comment by aaron-lane** [comment]
Hi @johanels. Thank you for your interest in the project!

We should be able to fulfill this request. Is there a particular use case which would be helpful to illustrate?

**Comment by johanels** [comment]
Good afternoon,

I've worked around this for now by stamping out a YAML file for the container launch, and then referencing it in the Google Compute Engine:

```
metadata {
    ssh-keys = "${var.ssh_username}:${file("${var.ssh_public_key}")}"
    gce-container-declaration = "${file("./container.yml")}"
  }
```

An in this example the YAML file is:

```
---
spec:
    containers:
      - name: nginx
        image: nginx
        env:
          - name: "TZ"
            value: "Europe/Amsterdam"
          - name: "GID"
            value: "1000"
          - name: "UID"
            value: "1000"
        stdin: false
        tty: false
        restartPolicy: Always
```

If it was possible to send these variables as part of the resource, I think it would be more elegant.

Regards,

**Comment by morgante** [comment]
Would you be interested in contributing this as a PR?

**Comment by aaron-lane** [comment]
Fixed by #19.
